src/main/resources/react4xp/_entries/RelatedArticles.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { Card, Text, Button } from '@statisticsnorway/ssb-component-library'
+import PropTypes from 'prop-types'
+
+class RelatedArticles extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isHidden: true
+    }
+
+    this.toggleBox = this.toggleBox.bind(this)
+  }
+
+  toggleBox() {
+    this.setState((prevState) => ({
+      isHidden: !prevState.isHidden
+    }))
+  }
+
+  getButtonBreakpoints() {
```

Comment:
Candidate for **DRY** ? also maybe prefer `else if` to multiple dangling `if`-s ? also reverse order of conditions which to avoid ambiguity of which condition should be chosen ? (it is not easy to know if it is `d-lg-flex` that should be returned or `d-flex` for a count of 5 from a quick look at the code!

```javascript
const getBreakpoints = (count, suffix) => {
  if (count > 6) {
    return `d-xl-${suffix}`
  } else if (count > 4) {
    return `d-lg-${suffix}`
  } else if (count > 3) {
    return `d-${suffix}`
  }

  return ''
}

// erstatt getButtonBreakpoints med 
getBreakpoints(relatedArticles.length, 'flex')

// erstatt getBreakpoints med
getBreakpoints(index, 'block')
```